toxencryptsave/crypto_pwhash_scryptsalsa208sha256/nosse/pwhash_scryptsalsa208sha256_nosse.c

```diff
@@ -257,11 +257,11 @@ escrypt_kdf_nosse(escrypt_local_t * local,
 		errno = EINVAL;
 		return -1;
 	}
-	if ((r > SIZE_MAX / 128 / p) ||
-#if SIZE_MAX / 256 <= UINT32_MAX
-	    (r > SIZE_MAX / 256) ||
-#endif
-	    (N > SIZE_MAX / 128 / r)) {
+	int test = (r > SIZE_MAX / 128 / p) || (N > SIZE_MAX / 128 / r);
```

Comment:
Use `_Bool`. Also rename to something more useful. `test` doesn't say anything. What is it testing?